lib/src/customer/customer.dart

```diff
@@ -89,6 +87,7 @@ class Customer {
 
   Future<void> _configureCustomerAsync() async {
     try {
+      await _loadCoiSettings();
```

Comment:
Why is this one needed?

Reply:
I can imagine: The settings aren't loadable if the notification "permission" wasn't granted during start, right?
We should review the notification on / off part once more and fix this globally, as there are more areas which are broken due to missing handling.